### DISCUSSION TITLE
Slack Channel for Fabric8 IO

### DISCUSSION THREAD

**Discussion by srinivasev** [fabric8io/kubernetes-client#4656] [Q&A]
Hi Team, 
Is there a Slack channel for Fabric8 IO ?.

**Comment by manusa** [comment]
Hi @srinivasev, do you mean in the Kubernetes Slack?

In the Kubernetes Slack we are listening on the Java channel. If you ask questions there, make sure to state that you're asking about the Fabric8 client (not the _"official"_ one).

We also have our own Gitter chat at https://gitter.im/fabric8io/kubernetes-client

**Reply by musabshak** [reply]
Could you send a link to this "Kubernetes Slack" channel?

**Reply by manusa** [reply]
kubernetes.slack.com
